libs/dal/src/repositories/change/change.schema.ts

```diff
@@ -48,4 +45,5 @@ changeSchema.virtual('user', {
 });
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export const Change = mongoose.models.Change || mongoose.model<IChangeDocument>('Change', changeSchema);
+export const Change =
+  (mongoose.models.Change as mongoose.Model<ChangeDBModel>) || mongoose.model<ChangeDBModel>('Change', changeSchema);
```

Comment:
I'm not sure why we have here "or", but I guess it's for the case when the model already is created.
So when the first part is true it means that the model is created so we cast it to the correct type.
The second part creates the model passing the correct type and schema.

Reply:
It is mainly for e2e testin for the exact reason you have mentioned 🙏